packages/manager/modules/anthos/src/dashboard/storage/remove/template.html

```diff
@@ -0,0 +1,31 @@
+<oui-modal
+    data-primary-action="$ctrl.removeStorage()"
+    data-primary-label="{{ ::'anthos_dashboard_remove_storage_submit' | translate }}"
+    data-primary-disabled="$ctrl.terminateConfirmation !== 'TERMINATE'"
```

Comment:
TERMINATE could be stored as a constant, Also if we're using an input we could benefit from `ng-pattern` instead

Reply:
This is done